src/pages/install/macos.md

```diff
@@ -0,0 +1,34 @@
+
+# Installing RGBDS on macOS
+
+RGBDS is available on [Homebrew](https://brew.sh) as the [`rgbds` package](https://formulae.brew.sh/formula/rgbds).
+
+```bash
+brew install rgbds
+```
+
```

Comment:
You can also install the [`master` branch](/docs/master) by passing the `--HEAD` flag:

```bash
brew install rgbds --HEAD
```

Reply:
Should this be mentioned at all here? I feel like it ought to go in the `master`-specific instructions.